StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1201UnitTests.cs

```diff
@@ -188,12 +232,22 @@ public class TestClass { }
     string this[string arg] { get; set; }
 }
 ";
-            var expected = new[]
-            {
-                this.CSharpDiagnostic().WithLocation(6, 12).WithArguments("indexer", "method")
-            };
+
+            var expected = this.CSharpDiagnostic().WithLocation(6, 12).WithArguments("indexer", "method");
 
             await this.VerifyCSharpDiagnosticAsync(testCode, expected, CancellationToken.None).ConfigureAwait(false);
+
+            string fixedCode = @"public interface OuterType
+{
+    event System.Action TestEvent;
+    string TestProperty { get; set; }
+    string this[string arg] { get; set; }
+    void TestMethod ();
```

Comment:
This test case could be improved by not having TestEvent at the correct location in the test code.